Dockerfile

```diff
@@ -1,10 +1,24 @@
-FROM golang:1.12
+FROM golang:1.12-alpine as build
 
 WORKDIR /src
 
+# build-base is required for tests
+RUN apk add --no-cache \
+    build-base \
+    git
+
 COPY go.mod go.sum ./
 RUN go mod download
 
 COPY . .
 RUN GO111MODULE=on go install .
 CMD ["whalebrew"]
+
+FROM alpine
+COPY --from=docker:18.06 /usr/local/bin/docker /usr/local/bin/docker
+COPY --from=build /go/bin/whalebrew /usr/local/bin/whalebrew
+
+ENTRYPOINT ["whalebrew"]
+LABEL io.whalebrew.config.volumes '["/var/run/docker.sock:/var/run/docker.sock", "$WHALEBREW_INSTALL_PATH:$WHALEBREW_INSTALL_PATH"]'
```

Comment:
Looks like there's no default `WHALEBREW_INSTALL_PATH` defined in the Dockerfile; should it be set to the default?

```Dockerfile
ENV WHALEBREW_INSTALL_PATH=/usr/local/bin
```

Reply:
`WHALEBREW_INSTALL_PATH` is to be evaluated on the host, not in the container, indeed, this could be reworked, we don't necessarily need to mount `WHALEBREW_INSTALL_PATH` on the host to `WHALEBREW_INSTALL_PATH` in the container, we could mount it to the default value of the install path, so that there is no need for env variable definition.